notebooks/finetuning/convfinqa_finetuning_wandb.ipynb

```diff
@@ -388,32 +388,39 @@
     "  ],\n",
     "  model=cfqa_finetune.finetuned_model.id+\"-ft\"\n",
     ")\n",
-    "print(\"#### Model response ####\")\n",
+    "print(\"#### Finetuned model response ####\")\n",
     "print(response.text)\n",
-    "print(\"########################\")"
+    "print(\"------\")"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "The response object is described in detail [here](https://docs.cohere.com/reference/chat#response). As you can see, the finetuned model responds in the DSL as expected and matches the ground truth. This DSL response can now be consumed by any downstream application or engine that can compute the final answer. For a comparison we show the base model response to the same inputs."
+    "As you can see, the finetuned model responds in the DSL as expected and matches the ground truth. This DSL response can now be consumed by any downstream application or engine that can compute the final answer. For comparison, we show the base model response to the same inputs."
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": 2,
+   "execution_count": 53,
```

Comment:
will the count show in the cookbook?

Reply:
i dont think so. i dont see it anywhere [here](https://cohere-preview-af0af6d1-01cd-49f9-8031-0273a26ab8d8.docs.buildwithfern.com/page/convfinqa-finetuning-wandb). i only see code line numbers